docs/architecture.html

```
@@ -0,0 +1,689 @@
+<title>stanpumpR — Architecture Map</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)